cmd/kraft/build/build.go

```diff
@@ -474,27 +475,52 @@ func (opts *Build) Run(cmd *cobra.Command, _ []string) error {
 			))
 		}
 
-		processes = append(processes, paraprogress.NewProcess(
-			fmt.Sprintf("building %s (%s)", targ.Name(), target.TargetPlatArchName(targ)),
-			func(ctx context.Context, w func(progress float64)) error {
-				err := opts.project.Build(
-					ctx,
-					targ, // Target-specific options
-					app.WithBuildProgressFunc(w),
-					app.WithBuildMakeOptions(append(mopts,
-						make.WithExecOptions(
-							exec.WithStdout(log.G(ctx).Writer()),
-							exec.WithStderr(log.G(ctx).WriterLevel(logrus.WarnLevel)),
-						),
-					)...),
-					app.WithBuildLogFile(opts.SaveBuildLog),
-				)
-				if err != nil {
-					return fmt.Errorf("build failed: %w", err)
-				}
-				return nil
-			},
-		))
+		if opts.Stats {
+			// Print lines of code
```

Comment:
Does `make print-loc` perform a build or does it not just do something fancy after a build?

Reply:
It invokes the build as a dependency, not sure if we can invoke it without actually building